Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+
+## global args
+ARG NODE_VERSION=18.16.1
+
+FROM node:${NODE_VERSION}
```

Comment:
have we tried using alpine image? that should reduce the image size by a considerable amount. not a blocker for this PR but we should check this.

Reply:
Thanks for the suggestion this made the image small by 1 GB. Tested and updated